setup.py

```diff
@@ -0,0 +1,44 @@
+from glob import glob
+from os.path import basename, splitext
+
+from setuptools import setup, find_packages
+
+with open("README.rst") as readme_file:
+    readme = readme_file.read()
+
+setup(
+    python_requires=">=3.5",
+    classifiers=[
+        "Development Status :: 2 - Pre-Alpha",
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: GNU Affero General Public License v3",
+        "Operating System :: Unix",
+        "Operating System :: POSIX",
+        "Natural Language :: English",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
```

Comment:
Does this actually support py3.6, py3.7?

Only 3.5 is supported at edx, while working to add support for 3.8.